cypress/support/commands.js

```diff
@@ -22,4 +22,13 @@
 //
 //
 // -- This will overwrite an existing command --
+
+
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import allMovies from "../fixtures/allMovies"
```

Comment:
Nice work using test data!

Maybe at some we can set this data as a fixture to make our tests even more compartmentalized.